_data/team.yml

```
@@ -256,3 +256,11 @@
   username: shiqi.mei
   photo: no
   country: China
+  
+- name: Eduardo Pereira
+  role: Head of Sales North America
+  department: Sales
+  github_username: epereira2020
+  username: Edu Pereira
+  photo: Yes
```

```suggestion
  photo: no
```

This needs to be no unless you have a photo in the team photos section